File: app/src/main/java/com/ikmr/banbara23/yaeyama_liner_checker/model/weather/Table.java
package com.ikmr.banbara23.yaeyama_liner_checker.model.weather;

/**
 * 時間毎の天気
 */
public class Table {

    private String hour;
    private String weather;
    private String windBlow;
    private String windSpeed;

    public void setHour(String hour) {
        this.hour = hour;
    }

    public String getHour() {
        return this.hour;
    }

    public void setWeather(String weather) {
        this.weather = weather;
    }

    public String getWeather() {
        return this.weather;
    }

    public void setWindBlow(String windBlow) {
        this.windBlow = windBlow;
    }

    public String getWindBlow() {
        return this.windBlow;
    }

    public void setWindSpeed(String windSpeed) {
        this.windSpeed = windSpeed;
    }

    public String getWindSpeed() {
        return this.windSpeed;
    }

    @Override
    public String toString() {
        return "Table{" +
                "hour='" + hour + '\'' +
                ", weather='" + weather + '\'' +
                ", windBlow='" + windBlow + '\'' +
                ", windSpeed='" + windSpeed + '\'' +
                '}';
    }
}

File: domain/src/main/kotlin/com/yaeyama/linerchecker/domain/weather/Table.kt
package com.yaeyama.linerchecker.domain.weather

/**
 * 時間毎の天気
 */
data class Table(
    /** 時刻（時） */
    val hour: String = "",
    /** 天気 */
    val weather: String = "",
    /** 風向き（Firebase のキーに合わせて名前は windBlow のまま） */
    val windBlow: String = "",
    /** 風速 */
    val windSpeed: String = "",
)
